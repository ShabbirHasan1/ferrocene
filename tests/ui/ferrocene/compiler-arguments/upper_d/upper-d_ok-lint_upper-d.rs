// check-fail
// compile-flags: -D exceeding_bitshifts -D

fn main() {}

// ferrocene-annotations: um_rustc_D
