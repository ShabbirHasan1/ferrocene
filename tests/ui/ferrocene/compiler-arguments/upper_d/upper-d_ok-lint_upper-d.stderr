error: Argument to option 'D' missing

